### ISSUE TITLE
A6:getting stuck in vae loss

### ISSUE THREAD

**Issue by imabackstabber** [imabackstabber/eecs498#2]
cannot figure out how to calculate the loss correctly...

**Comment by imabackstabber** [comment]
see [bce](https://pytorch.org/docs/stable/generated/torch.nn.functional.binary_cross_entropy.html) to correct mistake.
the default of "reduction" is "mean",set it to "sum" to correct it.
